js/sign/src/signers/integrity-block-signer.ts

```diff
@@ -18,41 +22,55 @@ type IntegritySignature = {
 };
 
 export class IntegrityBlockSigner {
+  // `webBundleId` is ignored if `is_v2` is false.
   constructor(
+    private readonly is_v2: boolean,
```

Comment:
Would it make sense to instead use an enum + switches and store the version in case more versions will be added in the future? Then the boolean wouldnt work anymore.

Reply:
Not really -- my intention is to eliminate everything related to v1 in a month :)